system/dummy_diag_publisher/CMakeLists.txt

```diff
@@ -4,15 +4,12 @@ project(dummy_diag_publisher)
 find_package(autoware_cmake REQUIRED)
 autoware_package()
 
-ament_auto_add_library(dummy_diag_publisher SHARED
-  src/dummy_diag_publisher_node/dummy_diag_publisher_node.cpp
-)
-
-rclcpp_components_register_node(dummy_diag_publisher
-  PLUGIN "DummyDiagPublisherNode"
-  EXECUTABLE dummy_diag_publisher_node
+ament_auto_add_executable(${PROJECT_NAME}
```

Comment:
Why did you change it to a normal ROS node?
It's generally recommended to use ComposableNode because users can control the process the node launches in.

I'd appreciate it if you could consider changing back to ComposableNode in another PR.

Reply:
OK, I will deal with it in another PR.